Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -72,6 +72,42 @@ public void InlineAutoDataSuppliesDataSpecimensOnlyForNonProvidedValues(string s
             Assert.NotEqual("bar", s3);
         }
 
+        // This test and its associated types is used to document one of the
+        // InlineAutoDataAttribute constructor overloads.
+        [Theory]
+        [MyCustomInlineAutoData(1337)]
+        [MyCustomInlineAutoData(1337, 7)]
```

Comment:
:+1: I think it'd be more clear for the test reader if we can do what @adamchester writes.